service/iam/api.go

```diff
@@ -790,7 +790,51 @@ func NewPermissions(client *client.DatabricksClient) *PermissionsAPI {
 }
 
 // Permissions API are used to create read, write, edit, update and manage
-// access for various users on different objects and endpoints.
+// access for various users on different objects and endpoints:
+//
+// * **[Cluster permissions](clusters)** — Manage which users can manage,
```

Comment:
run `make doc`. this may break the https://pkg.go.dev/github.com/databricks/databricks-sdk-go docs, as it points to non-existing links.

Reply:
![Screenshot 2023-08-03 at 13 21 23](https://github.com/databricks/databricks-sdk-go/assets/1850319/c1c5d7e4-51d0-40e6-8a24-cd8ba9cd0753)
No links to break if there are no links :)
Any prior art w.r.t. links within the REST API docs?

Reply:
From what I can tell, it is not possible to link to a service in our current OpenAPI model. I filed a ticket with @arusanov to support this. For now, I'm going to revert the links in this file

Reply:
yep, also in the openapi spec. this just breaks the logical flow of code comments.